frontend/src/components/buttons/CornerButton.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+	<!-- Without pb_bottom it's positioned on the top -->
+	<div class="page_button" :class="position">
+		<a href="link_target">{{ link_text }}</a>
+	</div>
+</template>
+
+<script>
+export default {
```

Comment:
https://vuejs.org/guide/typescript/overview.html#definecomponent

it is recommended to use this syntax for typescript (for all components)
```ts
export default defineComponent({
    prop: {},
})
```